main/main.c

```diff
@@ -639,12 +640,13 @@ static PHP_INI_MH(OnUpdateErrorLog)
 static PHP_INI_MH(OnUpdateMailLog)
 {
 	/* Only do the safemode/open_basedir check at runtime */
```

Comment:
small nit: safemode from comments should be removed, there is no longer such feature.